templates/Language.html.jinja2

```diff
@@ -14,9 +14,13 @@
     <tr>
       <th>Grammar File</th>
       <td>
+        {% if language.grammar_file %}
         <a href = {{ language.grammar_file }}>
           {{ m.show_grammar(language.grammar_file) }}
         </a>  
+        {% else %}
+          {{ m.show_grammar(language.grammar_file) }}
+        {% endif %}
```

Comment:
Unlabeled control end tag

*Origin: Jinja2Bear, Section: `jinja2`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpo67x_gqp/templates/Language.html.jinja2
+++ b/tmp/tmpo67x_gqp/templates/Language.html.jinja2
@@ -20,7 +20,7 @@
         </a>  
         {% else %}
           {{ m.show_grammar(language.grammar_file) }}
-        {% endif %}
+        {% endif %}{# if language.grammar_file #}
       </td>
     </tr>    
     <tr>
```

Reply:
Done